docs/src/pages/ui/theme/alternativeStyling/index.page.mdx

```diff
@@ -0,0 +1,501 @@
+</StyledView>;
+```
+
+#### Interactions
+
+There are various ways to customize an Amplify component and Styled Components will interact with these customizations in the following ways:
+
+##### Amplify Styling Props
+
+These are styling props that can be used directly on an Amplify component which will affect a single style property. Examples of styling props are `color` or `fontWeight`. These Amplify styling props will take precedence over Styled Component styling.
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { View } from '@aws-amplify/ui-react';
+
+const StyledView = styled(View)`
+  color: blue;
+`;
+
+<StyledView color="red">Using Styling props</StyledView>;
+```
+
+In the example above, the color of the view will be set to `red` because of the Amplify styling prop. [Here]() is a list of all Amplify styling props.
+
+##### Amplify variation props
+
+These are Amplify props that are added to certain Amplify components to change the look and/or behavior of the Amplify component. Examples of these variation props are `size` or `variation`.
+
+```jsx
+import React from 'react';
+import styled from "styled-components";
+import { Button } from '@aws-amplify/ui-react';
+
+const StyledButton = styled(Button)`
+  color: blue
+`;
+
+<StyledButton variation="primary'>Primary Button</StyledButton>
+```
+
+In the example above, the color of the button will be set to `white`, because the primary variation of the button has a color set to `white` and it takes precedence over the Styled Components styling. A list of Amplify variations can be found on [here]().
+
+##### Amplify Custom ClassNames
+
+These are custom classnames added onto Amplify components and can be used with CSS styling rules to modify the look of an Amplify component.
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { View } from '@aws-amplify/ui-react';
+
+const styledView = styled(View)`
+  color: blue;
+`;
+
+<styledView className="my-styled-view">My Styled View</styledView>;
+```
+
+```CSS
+/* External Style Sheet */
+.my-styled-view {
+  color: red;
+}
+```
+
+In the example above, the color of the view will be `blue` because the stlyed component will take precedence over the classname CSS.
+
+##### CSS Root Variable Overrides
+
+Amplify theming provides many CSS variables to customize the look and feel of the entire application. These CSS variables can be used at a root level to override all instances of component styling.
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '@aws-amplify/ui-react';
+
+const StyledButton = styled(Button)`
+  color: blue;
+`;
+
+<StyledButton>Button</StyledButton>;
+```
+
+```CSS
+/* External Style Sheet */
+:root {
+  --amplify-components-button-color: red;
+}
+```
+
+In the example above, the button color will be set to `blue` because the Styled Component will take precedence over the Amplify CSS variable.
```

Comment:
Similar question to above, is this true for all CSS variable overrides?

Reply:
This is true because of styling precedence, but unfortunately styling precedence isn't completely consistent because if they target the variables that are used in rules with a higher precedence then the variables will create changes that override the css in js changes. Some of these variables would be for variations or states (i.e. hover). I'm not sure what the best way to express this to the user would be.